site/third-party-projects.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: page
+title: Livy - Third-Party Projects
+description: Livy ecosystem projects
+group: nav-right
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+{% include JB/setup %}
+
+## {{ site.data.project.name }} Third-Party Projects
+
+This page lists external projects that add to Livy's ecosystem.
+Please open a pull request [here]({{ site.data.project.website_repository }}) if you would like to submit other projects.
```

Comment:
This should be `website_repository_mirror` so it points to the GitHub instead of the apache source.